<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemDefStringForm</class>
 <widget class="QWidget" name="ItemDefStringForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>52</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cbMultiline">
       <property name="text">
        <string>Multiline</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbSecure">
       <property name="text">
        <string>Secure</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
